prometheus-to-sd/translator/translator.go

```diff
@@ -84,6 +84,7 @@ func (t *TimeSeriesBuilder) Build() ([]*v3.TimeSeries, error) {
 	// metric is likely not to be whitelisted.
 	startTime := t.getStartTime(metricFamilies)
 	metricFamilies = filterWhitelisted(metricFamilies, t.config.SourceConfig.Whitelisted)
```

Comment:
Since there are two things to whitelist now, this should be renamed to filterWhitelistedMetrics.

Reply:
Thanks. Updated